internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -760,6 +760,19 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 									actionParams.(map[string]interface{})[logCustomFields] = newLogCustomFields
 								}
 							}
+							// check if our ruleset is of action 'skip'
+							if rules.([]interface{})[ruleCounter].(map[string]interface{})["action"] == "skip" {
+								// var newActionParams map[string][]string
+								for rule := range actionParams.(map[string]interface{}) {
+									for key, value := range actionParams.(map[string]interface{})[rule].(map[string]interface{}) {
+										var rulesList []string
+										for _, val := range value.([]interface{}) {
+											rulesList = append(rulesList, fmt.Sprintf("%s", val))
```

Comment:
do you need Sprintf here?

```suggestion
											rulesList = append(rulesList, val)
```

Reply:
This can work out as well: `rulesList = append(rulesList, val.(string))`